<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Groovy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="grpSearch">
      <item>
       <widget class="QLineEdit" name="txtSearch">
        <property name="text">
         <string>nine inch nails every day is exactly the same</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>&amp;Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tblSearchResults">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tblPlaylistView">
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="grpControls">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnPrev">
        <property name="text">
         <string>Pre&amp;vious</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="text">
         <string>&amp;Play/Pause</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string>&amp;Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="grpTime">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblElapsed">
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="songProgress">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTotal">
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Track">
    <property name="title">
     <string>&amp;Track</string>
    </property>
    <addaction name="action_Previous"/>
    <addaction name="action_Play_Pause"/>
    <addaction name="action_Next"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Track"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="text">
    <string>Pre&amp;vious</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="action_Play_Pause">
   <property name="text">
    <string>&amp;Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSearchButtonPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSearch</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSearchButtonPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblSearchResults</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>onQueueSong(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>songProgress</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>beginSeekSong(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>songProgress</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>endSeekSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNext</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPause</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPrev</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnPrev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Play_Pause</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Next</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Previous</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnPrev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSearchButtonPress()</slot>
  <slot>onQueueSong(QModelIndex)</slot>
  <slot>onPlaySong(QModelIndex)</slot>
  <slot>beginSeekSong(int)</slot>
  <slot>endSeekSong()</slot>
  <slot>songTick(qint64)</slot>
  <slot>songTotalTimeChanged(qint64)</slot>
  <slot>onBtnPause()</slot>
  <slot>onBtnNext()</slot>
  <slot>onBtnPrev()</slot>
 </slots>
</ui>
